lib/os.ml

```diff
@@ -232,6 +238,24 @@ let rm ~directory =
     Log.warn (fun f -> f "Failed to remove %s because %s" directory m);
     Lwt.return_unit
 
+let mv ~src dst =
+  let pp _ ppf = Fmt.pf ppf "[ MV ]" in
+  sudo_result ~pp:(pp "MV") ["mv"; src; dst ] >>= fun t ->
+  match t with
+  | Ok () -> Lwt.return_unit
+  | Error (`Msg m) ->
+    Log.warn (fun f -> f "Failed to move %s to %s because %s" src dst m);
+    Lwt.return_unit
```

Comment:
Perhaps, but if an error is returned here, it would be discarded in the build function (which called it).  `Os.mv` is called on the successful completion of the step - that is the important information.  Any `Os.mv` failure is an internal failure which will either show up when then the step/layer is reused or will be deleted at next time `ocluster-worker` starts.  The same appears in the other `_store.ml` drivers.

Reply:
I would find this reasoning more convincing if this were not a general-purpose seeming `Os` utility module, but I do understand now why this won't cause problems where it is actually be used. Thanks for the explanation!

Even if this is something worth changing at some point in a followup, given the fact that it follows the already implemented `rm` it seems reasonable for this change set.